docs/publishers/config.rst

```diff
@@ -450,6 +450,30 @@ loads.
                               // relative from the receiving iframe.
     }
 
+.. option:: sideBySide
+
+  ``Object``. This option lets you customize how the side-by-side mode behaves.
+
+  For example:
+
+  .. code-block:: javascript
+
+     window.hypothesisConfig = function () {
+       return {
+         sideBySide: {
+           mode: 'manual'
+         }
+       };
+     };
+
+  The following keys are supported in the :option:`sideBySide` object.
+
+  .. option:: mode
+
+    ``auto`` or ``manual``. Auto is the default value, where the main page content will be adjusted based on the sidebar
+    status, while manual indicates the guest app wants to take full control of handling side-by-side, which will
+    effectively disable that logic.
```

Comment:
It would be useful to provide some pointers here about how the web page is supposed to implement its own side-by-side logic (ie. using the `hypothesis:layoutchange`) event.

Reply:
I was planning to link to that event's documentation, with a brief explanation, but I switch to something else and forgot.